src/commands/synthetics/run-tests-lib.ts

```diff
@@ -192,52 +168,117 @@ const getTestListBySearchQuery = async (
   }))
 }
 
-export const getTestsList = async (
+export const getFinalVersionOfTestsFromCi = async (
   api: APIHelper,
   config: RunTestsCommandConfig,
   reporter: MainReporter,
-  suites: Suite[] = []
+  suites?: Suite[]
 ) => {
-  // If "testSearchQuery" is provided, always default to running it.
-  if (config.testSearchQuery) {
-    const testsToTriggerBySearchQuery = await getTestListBySearchQuery(api, config.global, config.testSearchQuery)
+  let testConfig: TriggerConfig[]
```

Comment:
```suggestion
  let testConfigs: TriggerConfig[]
```

Reply:
Or (as you have in the other function) `testsFromTestConfigs`

Reply:
will put it as `testConfigs` everywhere i think to keep it consistent 👍